api/src/pcapi/core/educational/api/offer.py

```diff
@@ -505,7 +512,10 @@ def edit_collective_offer_public(
             offer.domains = domains
         elif key == "educationalInstitutionId":
             if value:
```

Comment:
```suggestion
            if value is not None:
```
sinon on peut passer avec un `0` ce qui pourrait poser des problemes

Reply:
tu as oublié ce point